.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
```

Comment:
can we add a title here too like `[ENHANCEMENT]` if we keep the bug's title? Otherwise, we could also just stick to only tags

Reply:
im also thinking enhancement is different from feature. maybe we should have two separate templates for them? otherwise, I would just change this label from enhancement to feature

Reply:
yeah, I'll create a feature label now then